projects/igniteui-angular/src/lib/core/styles/components/grid/_grid-theme.scss

```diff
@@ -1310,4 +1374,9 @@
     %igx-grid__filtering-outlet {
         z-index: 99999;
     }
+
+    %igx-grid__row-editing-outlet {
+        z-index: 10000;
```

Comment:
@simeonoff Could you please confirm if this is okay?
We need the row-edit overlay to be above the pinned cells (z-index: 9999), below the filtering overlay (z-index: 99999) and below the virtual scrolls (z-index: 10001, [link](https://github.com/IgniteUI/igniteui-angular/blob/155ef109764d1d7bc5a27268ea1e64aaff6e60e1/projects/igniteui-angular/src/lib/core/styles/components/_common/_igx-vhelper.scss#L4))